### PR TITLE
Add support for gistifying and sending off to jsonblob

### PR DESCRIPTION
**Pull request by mejackreed**

### CODE REVIEW

File: lib/shimmy/json_blob.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      end
+    end
+
+    def blob_id
+      jsonify unless @blob
+      @blob.header['X-Jsonblob']
+    end
+
+    def blob_location
+      jsonify unless @blob
```
Comment (anarchivist) [inline lib/shimmy/json_blob.rb:7]: If a blob isn't created, create a blank one so it will mint a (predictable) URI

File: lib/shimmy/json_blob.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      @blob.header['Location'].gsub('http:', 'https:')
+    end
+
+    def update_blob
```
Comment (anarchivist) [inline lib/shimmy/json_blob.rb:15]: should return blob_location

**Comment by mejackreed** [comment]
Steps for this to work:
1. Create an instance of your shim `s = Shimmy::Shims::JscImageCollection.new`
2. Call `update_blob` on your shim

```
$ s.update_blog
=> "https://jsonblob.com/api/jsonBlob/560d3e36e4b01190df3a3ad7"
```

**Comment by anarchivist** [comment]
:rocket:

**Pull request merged**